src/compute/src/render/join/linear_join.rs

```diff
@@ -58,11 +76,21 @@ impl LinearJoinImpl {
         I: IntoIterator,
         I::Item: Data,
     {
-        match self {
-            Self::DifferentialDataflow => {
+        use LinearJoinImpl::*;
+        use YieldSpec::*;
+
+        match (self.implementation, self.yielding) {
```

Comment:
Nit: We cannot support `DifferentialDataflow` with yielding, but that's not expressed in the Rust types.

Reply:
I did this because it simplifies the two things separately in the `UpdateConfiguration` handler. Ideally we would also support configuring yielding of the DD join, but we need to wait for https://github.com/TimelyDataflow/differential-dataflow/pull/390 for that. I added a comment to that effect.